src/easy/handle.rs

```diff
@@ -1280,6 +1280,11 @@ impl Easy {
         self.inner.pipewait(wait)
     }
 
+    /// Same as [`Easy2::http_09`](struct.Easy2.html#method.http_09)
+    pub fn http_09(&mut self, allow: bool) -> Result<(), Error> {
```

Comment:
Nit: I might name this `http_09_allowed` which would more closely match the libcurl option name.